internal/pkg/plugin/trellogithub/workflow.go

```diff
@@ -34,8 +34,11 @@ type Options struct {
 }
 
 type Api struct {
-	Name            string
-	KanbanBoardName string
+	Name        string
```

Comment:
The name of the struct says "API" but according to the members of the struct, they don't seem like API related.